doc_source/cli-configure-completion.md

```diff
@@ -130,7 +123,7 @@ For the AWS completer to work successfully, you must first add it to your comput
 1. Add an export command at the end of your profile script that's similar to the following example\. Replace *`/usr/local/aws/bin`* with the folder that you discovered in the previous section\.
```

Comment:
Export would not be necessary for following the standard installation (already part of path)

Can cut sections or revise to relegate to optionally-navigable section headers